clang/test/SemaCXX/amdgpu-buffer-rsrc.cpp

```diff
@@ -0,0 +1,17 @@
+// REQUIRES: amdgpu-registered-target
+// RUN: %clang_cc1 -fsyntax-only -verify -std=gnu++11 -triple amdgcn -Wno-unused-value %s
+
```

Comment:
We probably want another similar sema test for OpenCL/HIP/OpenMP

Reply:
TBH I don't think there is any difference but I added them anyway.